src/styles/index.css

```diff
@@ -161,6 +165,20 @@
     --p-color-text-inverse: #171717;
     --p-color-text-link: #53B1FD;
 
+    /* Input */
+    --p-color-input-bg: var(--p-color-bg-0);
+    --p-color-input-border: #70707B;
+    --p-color-input-text: var(--p-color-text-default);
+    --p-color-input-placeholder: var(--p-color-text-subdued);
+
+    --p-color-input-bg-invalid: var(--p-color-input-bg);
+    --p-color-input-border-invalid: var(--p-color-sentiment-negative);
+    --p-color-input-text-invalid-icon: var(--p-color-sentiment-negative);
+
+    --p-color-input-activated-bg: var(--p-color-active);
+    --p-color-input-activated-border: var(--p-color-active);
+    --p-color-input-activated-text: var(--p-color-text-default);
```

Comment:
The activated input tokens are for checked checkbox/radios. Accidentally added those, can remove if we wanna.

Reply:
Can we swap that for `active` then? I believe that's the CSS semantic term instead of `activated`

Reply:
I think you're thinking of `checked`, which I can swap for that here.. I can't imagine another input type that would have a sort of activated state, so..

Reply:
Updated!

Reply:
Ah yeah you're right, sorry. Yea I'm mostly trying to make sure we can map tokens onto their semantic meanings for consistency. We'll be grateful for that later I think

Reply:
Totally! More than welcomed!